src/core/SliceRepresentation.js

```diff
@@ -33,7 +33,7 @@ export default class SliceRepresentation extends Component {
     this.lookupTable.applyColorMap(preset);
     this.piecewiseFunction = vtkPiecewiseFunction.newInstance();
     this.actor = vtkImageSlice.newInstance({ visibility: false });
-    this.mapper = vtkImageMapper.newInstance();
+    this.mapper = props.mapper ?? vtkImageMapper.newInstance();
```

Comment:
Would it make sense to have `props.mapper` be a string taking values: `{ "vtkImageMapper", "vtkImageArrayMapper" }` ? Then we can use the string value to instantiate this.mapper.

Reply:
Added a separate `mapperInstance` property

Reply:
It would make more sense for the mapper props to be set via a `mapperProps` property name-wise. Let's keep `props.mapper` as-is to avoid a breaking change.

Reply:
yes, I am keeping the original property `props.mapper` as-is, and added a new one for setting the mapper instance from outside. This can be removed once we have a `<Mapper>` component in the re-write.